package.json

```diff
@@ -7,83 +7,85 @@
     "  3. Go to http://localhost:8000",
     "================================================================================"
   ],
-  "name": "@alaskaairux/auro-avatar",
-  "version": "2.1.0",
-  "description": "Auro custom auro-avatar element",
-  "author": "Product design and research",
+  "name": "@aurodesignsystem/auro-avatar",
+  "version": "0.0.0",
```

Comment:
Version will get written during publish, right?

Reply:
The version number comes from the tag version. This gets committed back from semantic release.